base/deprecated.jl

```diff
@@ -1272,6 +1272,42 @@ export conv, conv2, deconv, filt, filt!, xcorr
 @deprecate_moved PollingFileWatcher "FileWatching" true true
 @deprecate_moved watch_file "FileWatching" true true
 @deprecate_moved FileMonitor "FileWatching" true true
+# FIXME : This causes problems with the `myid` in loading.jl
+#@deprecate_binding Distributed nothing true ", run `using Distributed` instead"
```

Comment:
See #24843 for a potential solution.

Reply:
Tried that but was causing AV build to fail for some reason. Have left a commented line in sysimg.jl for now.